velox/type/Type.h

```diff
@@ -45,6 +45,11 @@ namespace facebook::velox {
 
 using int128_t = __int128_t;
 
+struct __attribute__((__packed__)) Int96Timestamp {
```

Comment:
This is not common data structure and should be kept in parquet directiry

Reply:
Moved to PageReader.cpp. Thanks.